README.md

```diff
@@ -17,6 +17,13 @@ Configuration section below, and then run:
 
     helm install sematic-server sematic-ai/sematic-server -f /path/to/values.yml
 
+
+If you have already used Helm to install Sematic in your cluster, you can
+upgrade your installation to the latest version by running:
+
+    helm repo update
+    helm upgrade sematic-server sematic-ai/sematic-server -f /path/to/values.yml
+
```

Comment:
Please format the other CLI instructions in this doc in the same manner.
```suggestion
    ```bash
    $ helm repo update
    $ helm upgrade sematic-server sematic-ai/sematic-server -f /path/to/values.yml
    ```
```

Reply:
i actually dislike that format... the leading `$` just makes it a pain to copy and paste commands into your terminal.  but i'll add the rest of it